gulpfile.js

```diff
@@ -80,18 +113,48 @@ gulp.task('build-css', done => {
   done();
 });
 
-function runKarma(done) {
-  const karma = require('karma');
-  new karma.Server(
-    {
-      configFile: `${__dirname}/src/karma.config.js`,
-      ...karmaOptions,
-    },
-    done
-  ).start();
-}
-
 gulp.task(
   'test',
-  gulp.series('build-css', done => runKarma(done))
+  gulp.series('build-test-css', done => runKarma(done))
+);
+
+gulp.task('serve-pattern-library', () => {
+  servePatternLibrary();
+});
+
+// The following tasks bundle assets for the pattern library for use locally
+// during development. Bundled JS and CSS are not published with the package.
+
+// Bundle JS into local `build` folder
+gulp.task('bundle-js', () => {
+  return Promise.all(appBundles.map(config => createBundle(config)));
+});
+
+// Bundle SASS into local `build` folder
+gulp.task('bundle-css', function () {
+  fs.mkdirSync(STYLE_DIR, { recursive: true });
+  const bundles = cssBundles.map(entry =>
+    createStyleBundle({
+      input: entry,
+      output: `${STYLE_DIR}/${path.basename(entry, path.extname(entry))}.css`,
+      minify: IS_PRODUCTION_BUILD,
+    })
+  );
+  return Promise.all(bundles);
+});
+
+gulp.task(
+  'watch-css',
```

Comment:
I can see some confusion here because the watchers only rebuild into /build, but not for /lib as near as I can tell That means the pattern lib will live-load but not any consumers of the shared-package such as lms or the client.

We may want to ass a second set of watchers later on for this and migrate the scripts from package.json into gulp.

Reply:
> I can see some confusion here because the watchers only rebuild sass and js into /build, but not for /lib as near as I can tell That means the pattern lib will live-load but not any consumers of the shared-package such as lms or the client.

I'm sorry, I'm having trouble parsing this—could you possibly rephrase?

Reply:
Ignore me. Thinking more on this, because it's inside the context of `make dev`, it does only make sense that build/ gets updated with changes.